fs/tarfs.py

```diff
@@ -225,10 +225,9 @@ def getinfo(self, path, namespaces=None):
                 raise errors.ResourceNotFound(path)
 
 
-            raw_tar_info = member.get_info(
-                *([self.encoding, None]
-                if six.PY2 else [])
-            )
+            raw_tar_info = member.get_info(*(
+                [self.encoding, None] if six.PY2 else []
```

Comment:
This was so that coverage isn't different between Py2 and 3 right? I do the same thing.